### ISSUE TITLE
Can the connect room button be made available for sessions too?

### ISSUE THREAD

**Issue by pkoppenb** [indico/indico#5826]
**Is your feature request related to a problem? Please describe.**
In https://indico.cern.ch/event/1273211/timetable/?view=standard we have various rooms. The connect-room button for zoom only applies to the room defined at the top, which is not valid for the whole conference.

**Describe the solution you'd like**
I would like to have a button next to each room defined in each sessions, or alternatively a button valid for the considered time-slot (that's dangerous however).

**Describe alternatives you've considered**
What we did was to reset the location at the top each time we changed rooms. That caused some confusion about the location of the meeting.

**Comment by ThiefMaster** [comment]
Are you sure we don't show the connect button if you have a Zoom meeting actually assigned to the session/contribution? When checking your event, I see only a [single zoom meeting](https://indico.cern.ch/event/1273211/manage/videoconference/) linked to it.

**Comment by pkoppenb** [comment]
yes we have a single zoom meeting. Why should we have multiple? It's the same conference, just moving rooms at CERN. Having multiple may solve the connect room issue but then everyone outside would nee to disconnect and reconnect to another meeting, which also causes confusion (including people waiting in the wrong zoom room).

**Comment by ThiefMaster** [comment]
Ohh, you mean physical rooms. Sorry, misunderstood that.
